add-on/chrome/test/mochitest/browser_panel_privateBrowsing.js

```diff
@@ -0,0 +1,20 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+"use strict";
+
+add_task(function* test_panel_should_be_shown() {
+  yield LoopUI.openPanel();
+
+  Assert.notEqual(document.getElementById("loop-notification-panel"), null, "Panel exists");
+});
+
+add_task(function* test_init_copy_panel_private() {
+  let win = OpenBrowserWindow({ private: true });
+  yield new Promise(resolve => win.addEventListener("load", resolve));
+  yield win.LoopUI.openPanel();
+
+  Assert.equal(win.document.getElementById("loop-notification-panel"), null, "Panel doesn't exist for private browsing");
+
+  win.close();
+});
```

Comment:
You might want to switch to:

```js
let win = yield BrowserTestUtils.openNewBrowserWindow();
yield win.LoopUI.openPanel();
```
BrowserTestUtils has code for waiting for load etc.

You probably also want to change the `win.close()` to a cleanup function at the start of the test - this ensures it gets called even if the test fails, so you're less likely to impact other tests if something goes wrong (and the failures are clearer).

```js
  registerCleanupFunction(function* () {
    if (win) {
      yield BrowserTestUtils.close(win);
    }
  });
});